### PR TITLE
final PR minutes 10-29-2021

### PR DESCRIPTION
**Pull request by jcstein**
added audio recording to navigation. missing youtube link

### CODE REVIEW

File: town-hall/2021-10-29.md [hunk 1]
```diff
@@ -1,42 +1,116 @@
...
+## Mission, Values and Goals Snapshot Vote
 
 - Results of our first vote!
+- Community gave final word on [mission, values, goals](https://forum.developerdao.com/t/p-2-defining-our-mission-values-and-goals/137).
+    - 99.5% voted yes on [Snapshot](https://snapshot.org/#/devdao.eth/proposal/QmdZQD8h28PiWwwsdJo5mD2DBdC7BvYzrktmbsMoP4AcP6)
+
+## Season 0 Update
+
+- [Season 0 proposal](https://forum.developerdao.com/t/how-we-move-forward-as-a-dao-season-0/56/27) has received some great feedback as well, giving that more time to integrate into a final proposal. Please weigh in!
+- [Live Season 0 Proposal doc](https://docs.google.com/document/d/1zulImV8eZ_GuOiO-7qHl13vYbHMPiPVO4OF_U2vwG1g/edit) working proposal into a more refined overview of what a season is, guidelines, governance, organizational, etc.
```
Comment (kempsterrrr) [inline town-hall/2021-10-29.md:55]:
```suggestion
- [Live Season 0 Proposal doc](https://forum.developerdao.com/t/draft-proposal-season-0/219) working proposal into a more refined overview of what a season is, guidelines, governance, organizational, etc.
```

Comment (kempsterrrr) [reply town-hall/2021-10-29.md:55]: Swapping out Google Doc for discourse post

File: town-hall/2021-10-29.md [hunk 2]
```diff
@@ -1,42 +1,116 @@
...
+    - Pushing to a snapshot vote this week
+    - Season 0 = 2 months, with option to extend one month via snapshot vote if required
+        - this is a place for us all to figure it out and set the framework
+    - DAO Structure
+        - Governance Layer
+            - Core Team
+                - Guilds
+                    - Projects
+    - $DBUCKS will be the currency of reward for contributions in the DAO
```
Comment (kempsterrrr) [inline town-hall/2021-10-29.md:64]:
```suggestion
    - $DBUCKS will eventually become the native governance token for Developer DAO distributed to members based on their contributions
```

Comment (kempsterrrr) [reply town-hall/2021-10-29.md:64]: Switching up the language to make it a little more accurate regarding positioning of token

File: town-hall/2021-10-29.md [hunk 3]
```diff
@@ -1,42 +1,116 @@
...
+    - Announcements will be made when voting is open on these topics.
+    - Seed Club could add value to organization of DAO - conversation is still open
+
+
+## Partnerships
+
+- Working on partnership with [Gitcoin](https://gitcoin.co) and [Moonshot Collective](https://moonshotcollective.space/)
+    - [Austin Griffith - scaffold-eth](https://austingriffith.com/portfolio/scaffoldeth/)
+- In discussion to work with Radicle for possible token exchange. More discussion required.
+    - [Radicle](https://radicle.xyz) is interested in forming a partnership
```
Comment (kempsterrrr) [inline town-hall/2021-10-29.md:76]:
```suggestion
- In discussion with [Radicle](https://radicle.xyz) for a potential partnership but conversations at an early stage. Will update as this progresses.
```

File: town-hall/2021-10-29.md [hunk 4]
```diff
@@ -1,42 +1,116 @@
...
+- Look for low-lift moves as we are still in the very early stages
```
Comment (kempsterrrr) [inline town-hall/2021-10-29.md:77]:
```suggestion
- For anyone that has connections please feel free to make them but aim for a low-lift partnership to being with such as coordinating on events or workshops, whilst we're getting more organised
```

File: town-hall/2021-10-29.md [hunk 5]
```diff
@@ -1,42 +1,116 @@
...
-## Season 0
+- `nathang.eth` gives update
+    - Two use cases
+        - Notion - DAO Wiki
+            - Internal tool, creating structure to it to solicit contributions from users on content and design
+        - [Airtable](https://airtable.com/shrqw4chX4AuTCdf3/tbljejdzelezqT0W7) - Gathering Resources to share within our community, organizing so members and public can get a better understanding of DevDAO
+            - Will host the resources database, `kempsterrrr` leading the charge on this
+            - sync to airtable with markdown
+- Discord bot
+    - create bot to use / command to add resources to databases `NoahH`
```
Comment (kempsterrrr) [inline town-hall/2021-10-29.md:89]:
```suggestion
        - [Airtable](https://airtable.com/shrqw4chX4AuTCdf3/tbljejdzelezqT0W7) - Gathering and organising learning resources that are shared within our community in Airtable so they are more discoverable and easily shared with people inside and outside the DAO. Our first public good.
            - Will host the resources database public and contributions guidelines internally, `kempsterrrr` leading the charge on this
            - part of this project is auto-generating our [resources GitHub repo](https://github.com/Developer-DAO/resources) via a Github action (similar to "awesome-xxx" repo's) so to provide web3 learning resource on GitHub
            - Also creating a Discord bot go add new resources to the resource base using `/commands`, currently led by`NoahH`
            - Reach out in [#resource-base](https://discord.gg/CVyB5xW2pH) on discord if interested in contributing to any of element of this project
```

File: town-hall/2021-10-29.md [hunk 6]
```diff
@@ -1,42 +1,116 @@
...
+## D_D Merch
+
+- Merch was mentioned in chat this last town hall. Potentially something for designers in the DAO to drum up some ideas. A `#merch` channel was created after last town hall.
+- [MetaFactory](https://www.metafactory.ai/) has been mentioned as an on-chain provider of merch
 
 ## Notes
 
+- How should we Contribute? What projects are working? What projects are down the pipeline?
+    - DAO Wiki should help to serve this purpose as it is developed
+- Want to open up a new discussion? [Open a discussion on the site repo](https://github.com/Developer-DAO/developerdao.com/discussions/new)
```
Comment (kempsterrrr) [inline town-hall/2021-10-29.md:101]:
```suggestion
- Want to open up a new discussion ? Try to find the most relevant channel in discord and ask away! If discussions become significant, either open a discord thread, request mods to open a channel or post a thread on general category of discord (provide as much context as you can)
- What to start a new project? Pitch your idea in [#new-project-pitches](https://discord.gg/tYEYb2p22C) on discord and feel free to open a discord thread if there's interest and you want a place to discuss further. If you feel you have a good level of interest request mods to open up a channel where the team can work. Will be providing more detail on project processes in Season 0. For now, feel free to build.
```

File: town-hall/2021-10-29.md [hunk 7]
```diff
@@ -1,42 +1,116 @@
...
+
 ## Attendees
 
-## Video/Audio Recording
+- `willblackburn` sent out [POAP](https://poap.xyz/) to attendees
+
+## Action Items
+
+- Each DAO project has plenty to do! Reach out in their respective channels on Discord.
+- Take a look at [Draft Proposal](https://docs.google.com/document/d/1zulImV8eZ_GuOiO-7qHl13vYbHMPiPVO4OF_U2vwG1g/edit)
```
Comment (kempsterrrr) [inline town-hall/2021-10-29.md:110]:
```suggestion
- Take a look at [Draft Proposal](https://forum.developerdao.com/t/draft-proposal-season-0/219)
```

Comment (kempsterrrr) [reply town-hall/2021-10-29.md:110]: Replacing with discourse link.

**Review by kempsterrrr — APPROVED** [review]
Smashed it doing these notes @jcstein can't thank you enough 👍🏻 

Made some amends and replaced some links. If you're happy with these commit them in and I'll merge them first thing tomorrow :)

**Comment by gjsyme** [comment]
Sorry it took me a bit to get the video up: https://youtu.be/mHuURlNWx1s

**Comment by jcstein** [comment]
not a problem at all! @gjsyme

File: town-hall/2021-10-29.md [hunk 8]
```diff
@@ -1,42 +1,118 @@
...
+- Weigh in with feedback on [Season 0 proposal](https://forum.developerdao.com/t/how-we-move-forward-as-a-dao-season-0/56/27) 
+    - Pushing to vote this week
+
+## Audio Recording
+
+[YouTube audio](https://www.youtube.com/watch?v=)
```
Comment (jcstein) [inline town-hall/2021-10-29.md:116]: https://youtu.be/mHuURlNWx1s

**Review by jcstein — COMMENTED** [review]
adding youtube link

**Pull request merged**